tools/Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM python:3.10-slim-bullseye
+MAINTAINER Opentrons Engineering <engineering@opentrons.com>
+
+ENV BUILDENV_ROOT=/build-environment
+ENV POETRY_HOME=/build-environment/poetry
+ENV POETRY_VERSION=1.2.1
+ENV PATH=$PATH:/build-environment/poetry/bin
+ENV POETRY_VIRTUALENVS_IN_PROJECT=true
+ENV HOME=/build-environment
+
+RUN apt update && apt install -y curl wget file git
+RUN rm /bin/sh && ln -s /bin/bash /bin/sh
+RUN mkdir /build-environment
+ADD . /build-environment/tools
+
+# TODO: This should be a release version of the SDK, but one does not
+# exist yet.
+RUN \
```

Comment:
Also if you separate this into it's own build stage, you can download this while running everything else if you use [docker buildx build](https://docs.docker.com/engine/reference/commandline/buildx_build/) then use a [COPY FROM command](https://docs.docker.com/build/building/multi-stage/#name-your-build-stages).